backend/settings/base.py

```diff
@@ -149,3 +164,35 @@
     'ROTATE_REFRESH_TOKENS': False,
     'BLACKLIST_AFTER_ROTATION': True,
 }
+
+# Django CORS headers configuration
+# https://pypi.org/project/django-cors-headers/
+CORS_ORIGIN_WHITELIST = [
+    'http://localhost:3000'
```

Comment:
I think this should be an env var as well,  just in case another person decides to run on a different port.

Reply:
okay